src/ReactHooks/useCharSelectionCallback.tsx

```diff
@@ -26,7 +26,10 @@ export default function useCharSelectionCallback() {
         const weaponId = database.createWeapon(weapon)
         database.setWeaponLocation(weaponId, characterKey)
       }
-      history.push(`/character/${characterKey}`)
+      if (!tabName) {
+        tabName = "overview"
```

Comment:
should keep the `overview` as the default path, so it should be `/character/XXXX` instead of `/character/XXXX/overview`